tests/sentry/tasks/test_post_process.py

```diff
@@ -1502,6 +1506,86 @@ def test_does_not_skip_when_is_new(self, mock_get_commit_context):
         )
 
 
+class SnoozeTestSkipSnoozeMixin(BasePostProgressGroupMixin):
+    @with_feature("organizations:escalating-issues")
+    @patch("sentry.signals.issue_escalating.send_robust")
+    @patch("sentry.signals.issue_unignored.send_robust")
+    @patch("sentry.rules.processor.RuleProcessor")
+    @with_feature("organizations:issue-platform-api-crons-sd")
+    def test_invalidates_snooze_ff_on(
+        self, mock_processor, mock_send_unignored_robust, mock_send_escalating_robust
+    ):
+        event = self.create_event(data={"message": "testing"}, project_id=self.project.id)
+        group = event.group
+        can_generate_escalating_forecasts = group.issue_type.can_generate_escalating_forecasts(
+            self.project.organization
+        )
+
+        # Check for has_reappeared=False if is_new=True
+        self.call_post_process_group(
+            is_new=True,
+            is_regression=False,
+            is_new_group_environment=True,
+            event=event,
+        )
+
+        assert GroupInbox.objects.filter(group=group, reason=GroupInboxReason.NEW.value).exists()
+        GroupInbox.objects.filter(group=group).delete()  # Delete so it creates the UNIGNORED entry.
+        Activity.objects.filter(group=group).delete()
+        mock_processor.assert_called_with(EventMatcher(event), True, False, True, False)
+
+        event = self.create_event(data={"message": "testing"}, project_id=self.project.id)
```

Comment:
Do we need this chunk?

Reply:
I have this function mimicking `test_invalidates_snooze` in SnoozeTestMixin so that it verifies that Error and Performance issues are unaffected when the feature flag is enabled.

Reply:
I see - is there a reason we've put this in `SnoozeTestSkipSnoozeMixin` instead of adding it to `SnoozeTestMixin`? Once we remove the flags, this should be the expected behavior overall.

Reply:
I have them separate at the moment since the way that create_event is mocked in PostProcessGroupAggregateEventTest, does not pass the SnoozeTestMixin tests. Definitely something that needs to be addressed later on.